Makefile

```diff
@@ -4,19 +4,17 @@ all: clean dir wasm
 
 wasm:
 	$(CC) yoga/yoga/*.cpp bindings/*.cc \
-		--bind -Os --memory-init-file 0 --llvm-lto 1 \
+		--bind \
 		-Iyoga \
 		-g0 \
-		-O2 \
+		-Os \
 		-flto \
 		--closure 1 \
-		-msimd128 \
 		-s WASM=1 \
 		-s WASM_ASYNC_COMPILATION=1 \
-		-s EXTRA_EXPORTED_RUNTIME_METHODS=['cwrap','setValue'] \
```

Comment:
`EXTRA_EXPORTED_RUNTIME_METHODS` is obsolete method and I can't find any information on why it is here so

@shuding why did you add this param, do I need to return it back?

Reply:
Sorry I can't recall - guess I saw it somewhere from emcc's issues. I guess we can remove it!